app/src/pages/inside/settingsPage/settingsTabs/notificationsTab/notificationsTab.jsx

```diff
@@ -55,38 +56,51 @@ export class NotificationsTab extends Component {
       update(newConfig);
     }
   };
+
   isAbleToEditForm = () =>
     canConfigreEmailNotifications(this.props.userRole, this.props.projectRole);
-  isAbleToSubmit = ({ emailCases }) => emailCases.every(({ confirmed }) => confirmed);
-  convertEmailCaseForSubmission = (obj) => {
-    const { informOwner, launchNames, recipients, sendCase, tags } = obj;
+
+  isAbleToSubmit = ({ rules }) => rules.every(({ confirmed }) => confirmed);
+
+  convertRulesForSubmission = (obj) => {
```

Comment:
`convertRuleForSubmission ` - looks like this function converts only one rule

Reply:
Fixed